src/phase0/state_transition/epoch_processing.rs

```diff
@@ -282,8 +282,13 @@ pub fn process_randao_mixes_reset<
         MAX_VALIDATORS_PER_COMMITTEE,
         PENDING_ATTESTATIONS_BOUND,
     >,
-    _context: &Context,
+    context: &Context,
 ) -> Result<(), Error> {
+    let current_epoch = get_current_epoch(state, context);
+    let next_epoch = get_current_epoch(state, context) + 1;
```

Comment:
can save the overhead of a fn call here
```suggestion
    let next_epoch = current_epoch + 1;
```